src/styles/calendar.scss

```diff
@@ -0,0 +1,60 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  margin: $calendar-margin;
+  padding: $calendar-padding;
+  width: $calendar-width;
+  gap: $cell-gap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include square($cell-size);
+    @include font-style ($font-family, $font-size);
+    cursor: pointer;
+    
+    border: 1px solid black;
+    border-radius: 5%;
+    background-color: #eee;
+
+    transition: transform 0.3s,
+      background-color 0.2s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      box-shadow: 4px 4px teal;
+    }
+  
+    @for $index from 1 to 32 {
+      &:nth-child(#{$index}) {
+  
+        &::before {
+          content: "#{$index}";
+        }
+      }
+    }
+  }
+
```

Comment:
use nesting

Reply:
unfortunately i have no clue what are you talking about here at all((((